### PR TITLE
#261 260 bug filter does not work by date and data

### PR DESCRIPTION
**Pull request by VolodjaTur**
Fix 261 and 260 bugs.

### CODE REVIEW

**Comment by RostyslavKhasanov** [comment]
Check Travis build

**Comment by ZakharSkaletskyi** [comment]
You have 1 Checkstyle violation. 
[ERROR] src/main/java/greencity/repository/options/UserFilter.java:[64] (javadoc) SummaryJavadoc: First sentence of Javadoc is missing an ending period.

**Comment by ZakharSkaletskyi** [comment]
Please? try to build project before every pull request.

File: src/main/java/greencity/repository/options/UserFilter.java [hunk 1]
```diff
@@ -52,11 +52,28 @@ public Predicate toPredicate(Root<User> root, CriteriaQuery<?> criteriaQuery, Cr
...
         return cb.or(
             cb.like(r.get(RepoConstants.FIRST_NAME), reg),
             cb.like(r.get(RepoConstants.LAST_NAME), reg),
             cb.like(r.get(RepoConstants.EMAIL), reg),
             cb.like(r.get(RepoConstants.REGISTRATION_DATE).as(String.class), reg)
         );
     }
+
+    /**
+     * Returns a String criteria for search
```
Comment (ZakharSkaletskyi) [inline src/main/java/greencity/repository/options/UserFilter.java]: In the end must be dot.

Comment (VolodjaTur) [reply src/main/java/greencity/repository/options/UserFilter.java]: Check again please

**Pull request merged**